test/ConfigDiscoveryTest.php

```diff
@@ -87,24 +95,24 @@ public function createModulesConfig()
     public function assertOptionsContainsNoopInjector(Collection $options)
     {
         if ($options->isEmpty()) {
-            throw new ExpectationFailedException('Options array is empty; no NoopInjector found!');
+            $this->fail('Options array is empty; no NoopInjector found!');
```

Comment:
When writing custom assertions, the PHPUnit documentation recommends throwing the above exception over using `fail()`. (Similarly, it recommends `assertThat()` vs the other assertion methods.)

I'll revert on merge.